.github/workflows/c-cpp.yml

```diff
@@ -60,9 +60,30 @@ jobs:
           cp build/ntsc-final-port/pd.exe bin/pd
           touch bin/data/put_your_rom_here.txt
       - name: Upload artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: pd-i686-linux
           path: bin/
           retention-days: 0
-      
+
+  publish-latest-build:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/port'
+    needs: [build-i686-windows, build-i686-linux]
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Holdoff.
Checkout shouldn't be necessary. Going to force push over this once CI succeeds in the fork.

Reply:
Disregard. I wasn't thinking.
We don't need the repo to publish the assets but we do to push the tags.